src/orders/components/OrderDraftDetails/OrderDraftDetails.test.tsx

```diff
@@ -0,0 +1,117 @@
+import { channelsList } from "@dashboard/channels/fixtures";
+import { channelUsabilityData, order } from "@dashboard/orders/fixtures";
+import Wrapper from "@test/wrapper";
+import { render, screen } from "@testing-library/react";
+
+import OrderDraftDetails from "./OrderDraftDetails";
+
+// Mock the child component to avoid needing complex context providers
+jest.mock("../OrderDraftDetailsProducts/OrderDraftDetailsProducts", () => ({
+  __esModule: true,
+  default: () => <div data-testid="mock-order-draft-details-products" />,
+}));
```

Comment:
if we need to do this, then we probably should refactor the component 😅 

ideally we should only mock context **provider**, not children

Reply:
Let's park it for another PR.